core/src/workflows/hold_entry.rs

```diff
@@ -27,13 +28,24 @@ pub async fn hold_entry_workflow<'a>(
             let message = "Could not get validation package from source! -> Add to pending...";
             context.log(format!("debug/workflow/hold_entry: {}", message));
             context.log(format!("debug/workflow/hold_entry: Error was: {:?}", err));
-            add_pending_validation(entry_with_header.to_owned(), Vec::new(), &context);
+            add_pending_validation(
+                entry_with_header.to_owned(),
+                Vec::new(),
+                ValidatingWorkflow::HoldEntry,
+                &context,
+            );
             HolochainError::ValidationPending
         })?;
     let validation_package = maybe_validation_package.ok_or({
-        let message = "Source did respond to request but did not deliver validation package! This is weird! Entry is not valid!";
+        let message = "Source did respond to request but did not deliver validation package! (Empty response) This is weird! Let's try this again later -> Add to pending";
         context.log(format!("debug/workflow/hold_entry: {}", message));
-        HolochainError::ValidationFailed("Entry not backed by source".to_string())
+        add_pending_validation(
+            entry_with_header.to_owned(),
+            Vec::new(),
+            ValidatingWorkflow::HoldEntry,
+            &context,
+        );
```

Comment:
If you use `ok_or`, it's going to run this every time. You probably want `ok_or_else` to only run this in the Err case, right?

Reply:
Jeez, yes! :see_no_evil: 
Thanks, @maackle :pray: